doc/lsst.ap.verify/running.rst

```diff
@@ -0,0 +1,80 @@
+.. _ap-verify-running:
+
+#######################################
+Running ap_verify From the Command Line
+#######################################
+
+``ap_verify`` is a Python script designed to be run on both developer machines and verification servers.
+This page describes the minimum options needed to run ``ap_verify``.
+For more details, see the :ref:`ap-verify-cmd` or run ``ap_verify`` with the ``-h`` argument.
+
+.. _ap-verify-dataset-name:
+
+Datasets as Input Arguments
+---------------------------
+
+While ``ap_verify`` is not a :ref:`command-line task<lsst.pipe.base>`, the command-line interface is designed to resemble that of command-line tasks where practical.
+The most important difference is that, since ``ap_verify`` begins with an uningested :ref:`dataset<ap-verify-datasets>`, the input argument is a dataset name rather than a repository.
```

Comment:
```
Since ``ap_verify`` begins with an uningested :ref:`dataset<ap-verify-datasets>`, the input argument is a dataset name rather than a repository.
```